languages/javascript/exercises/concept/errors/.docs/instructions.md

```diff
@@ -0,0 +1,95 @@
+Elena is the new quality manager of a newspaper factory. As she has just arrived in the company, she has decided to review some of the processes in the factory to see what could be improved. She found out that technicians are doing a lot of quality checks by hand. She sees there a good oportunity for automation and ask you, a freelance developer, to develop a piece of software to monitor some of the machines.
```

Comment:
She sees there a good -> She sees there **is** a good

Reply:
Isn't it possible to say both "She sees there a good oportunity" and "She sees there is a good oportunity" ?

Reply:
aaa7fe2a557b19ad60f0a2626d6e2f7b734d1f0c